Look and feel updates

sharifsalah opened the pull request:
Replaced the dancing robots PNG file with a compressed JPG version that's the same dimensions but a much smaller file size to speed load time.

Also increased the left and right margin for most of the text on the landing page a little. 

Staged at:

https://5cd5a49e613948000997bc84--elated-haibt-1b47ff.netlify.com/

bcipriano commented on assets/scss/_variables_project.scss:
```diff
@@ -24,6 +24,10 @@ $navbar-dark-disabled-color: rgba($black, 0.25);
+  
+p {
+  margin-left: 1em;
+  margin-right: 1em; }
```
nit: I think that it's standard practice to put the closing bracket on its own line, like in the block below this one. I'm not a CSS expert though.

sharifsalah replied:
Thanks, this was bugging me as well, so I've tidied up the SCSS in this file. It's looking equivalent to me, but PTAL if you have a couple of minutes.

sharifsalah replied:
Staged at:

https://5cdb8ad88a8ede00087f96f0--elated-haibt-1b47ff.netlify.com/

bcipriano commented on assets/scss/_variables_project.scss:
```diff
@@ -49,6 +53,8 @@ $navbar-dark-disabled-color: rgba($black, 0.25);
     font-weight: 500;
     line-height: 1.2;
+    margin-left: 0;
+    margin-right: 0;
```
Not really new in this PR, but if this is scss format you should be able to nest these `.banner` child blocks inside the main `.banner` block, like:

```
.banner {
  margin: 3em 1em;
  text-align: center;

  > .banner-image {
    margin-bottom: 2em; 
    margin-left: auto;
    margin-right: auto;
    max-width: 65%;
  }

  > .banner-text {
    /* and so on */
} /* then close the main .banner block */
```

sharifsalah replied:
I've updated the file, so PTAL.

bcipriano commented on assets/scss/_variables_project.scss:
```diff
@@ -23,68 +23,84 @@ $navbar-dark-disabled-color: rgba($black, 0.25);
-    width: 100%;
-  }
+
+.banner-image img {
```
Same down here, this can be nested within the `.banner-image` block above.

sharifsalah replied:
Done.

bcipriano commented on assets/scss/_variables_project.scss:
```diff
@@ -23,68 +23,84 @@ $navbar-dark-disabled-color: rgba($black, 0.25);
-      text-align: left;
-    }
-  }
+	.banner {
```
Leading whitespace in these next few blocks looks a bit off here compared to the rest of the file, is this intentional?

sharifsalah replied:
Hmmm, that's very strange, I was trying out using another text editor to update the SCSS, so it must be something to do with that.

I fixed this in VI... thanks for spotting!

sharifsalah commented:
Thanks Brian, I appreciate the nitpicky style comments.

I've addressed your comments. The changes are staged at:

https://5cdcad00c832b50007ba827c--elated-haibt-1b47ff.netlify.com/

I'll go ahead and merge this, but I'm going to follow up with a new PR to add the ASWF logo that will probably touch the SCSS again, so we can address any further issues there.
